frontend/src/component/pages/Home.tsx

```diff
@@ -69,6 +72,7 @@ export class Home extends Component<Props, State> {
   signInModal = React.createRef<SignInModal>();
   createShortLinkSection = React.createRef<CreateShortLinkSection>();
   changeLogModalRef = React.createRef<ChangeLogModal>();
+  toastRef = React.createRef<Toast>();
```

Comment:
For your information: Eventually, we will create a [Eventbus](https://gameprogrammingpatterns.com/event-queue.html) to allow any part of our systems to show popup without knowing how the rest of the system works.

Reply:
Nice!